blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/serializers/metrics/BaseRollupSerDes.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2016 Rackspace
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.rackspacecloud.blueflood.io.serializers.metrics;
+
+import com.codahale.metrics.Histogram;
+import com.google.protobuf.CodedInputStream;
+import com.google.protobuf.CodedOutputStream;
+import com.rackspacecloud.blueflood.exceptions.SerializationException;
+import com.rackspacecloud.blueflood.io.Constants;
+import com.rackspacecloud.blueflood.types.BaseRollup;
+import com.rackspacecloud.blueflood.utils.Metrics;
+
+import java.io.IOException;
+
+/**
+ * The serialization and deserialization methods for sub-metrics used by BasicMetricRollup and GaugeMetricRollup.
+ * Comment sub-metrics are:
+ * <ul>
+ *     <li>average</li>
+ *     <li>max</li>
+ *     <li>min</li>
+ *     <li>sum</li>
+ * </ul>
+ */
+public class BaseRollupSerDes extends AbstractSerDes {
+
+    /**
+     * Our internal metric to count the number of rollups
+     */
+    protected static Histogram rollupSize = Metrics.histogram( BasicRollupSerDes.class, "Rollup Metric Size" );
```

Comment:
Comment seems wrong. It's not a count of number of rollups. It seems to be tracking the buffer size. But that could be existing comment. 

I wonder if this metric is useful at all. Doesn't the buffer size the same for all? I mean maybe there will be a difference between Gauge rollup buffer size and Basic metric rollup buffer size, now that we add "sum" for Basic, but not Gauge. 

You don't need to change anything in the code... I'm just commenting :-)